public/svgs/forward-arrow.svg

```diff
@@ -0,0 +1 @@
+<svg height='100px' width='100px'  fill="#26529a" xmlns="http://www.w3.org/2000/svg" data-name="Layer 1" viewBox="0 0 100 100" x="0px" y="0px"><title>Forward</title><path d="M67.34,15.9a3.55,3.55,0,1,0-5,5L87.85,46.44H3.53a3.56,3.56,0,0,0,0,7.11H87.85L62.3,79.06a3.62,3.62,0,0,0,0,5,3.53,3.53,0,0,0,5,0l31.6-31.6a3.47,3.47,0,0,0,0-5Z"></path></svg>
```

Comment:
It looks like we can bake both this and the `back-arrow` into the `utilities/svgs` so that we don't have to carry these files and can instead go through `Icon` to render them. Lemme know if that all gives you any trouble.

Reply:
both moved in to `utilities/svgs` and code updated accordingly